back/engines/free/email_campaigns/app/models/email_campaigns/campaigns/admin_digest.rb

```diff
@@ -142,13 +132,8 @@ def days_ago
       ((t2 - t1) / 1.day).days
     end
 
-    def stat_increase(stats = [])
-      second_last_agos = stats.select { |t| t > (Time.now - (days_ago * 2)) }
-      last_agos = second_last_agos.select { |t| t > (Time.now - days_ago) }
-      {
-        increase: last_agos.size,
-        past_increase: second_last_agos.size
-      }
+    def stat_increase(dates = [])
```

Comment:
Understand what this variable is for now, but it wasn't obvious initially so maybe `stat_dates` to be a little clearer?